codespell_lib/data/dictionary.txt

```diff
@@ -1303,16 +1303,34 @@ alignrigh->alignright
 alikes->alike, likes,
 aline->align, a line, line, saline,
 alined->aligned
+aling->align, along, a line, ailing, sling,
+alinged->aligned
+alinging->aligning
 alingment->alignment
+alings->aligns, slings,
 alinment->alignment
 alinments->alignments
 alising->aliasing
 aliver->alive, liver, a liver, sliver,
 allcate->allocate
+allcateing->allocating
+allcater->allocator
+allcaters->allocators
+allcating->allocating
 allcation->allocation
 allcator->allocator
+allcoate->allocate
 allcoated->allocated
-allcommnads->allcommands
+allcoateing->allocating
+allcoateng->allocating
+allcoater->allocator
+allcoaters->allocators
+allcoating->allocating
+allcoation->allocation
+allcoator->allocator
+allcoator->allocator
```

Comment:
`allcoator` is duplicated too :confused:

Reply:
I almost checked for more duplicates! 😢 

Thank you @lurch.  This should be all.  This has been a very involved PR! 😄  Thank you for all of your support!

Reply:
Next time I open a PR I'll work locally so I can run `make check` and avoid all of this trouble!! 🤣